creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -252,7 +253,9 @@ class SnapbackSM {
         wallet: [userWallet],
         creator_node_endpoint: primaryEndpoint,
         // Note - `sync_type` param is only used for logging by nodeSync.js
-        sync_type: syncType
+        sync_type: syncType,
+        // immediate = true will ensure secondary skips debounce and evaluates sync immediately
+        immediate
```

Comment:
without this the code was still getting debounced and taking way too long

Reply:
but doesn't this bypass the new bull queue. also isn't the debounce 0 or did we have some small value?

Reply:
it does bypass it. the debounce is set to 5s locally. even though we set it to 0 in config.js i realized that isn't getting respected bc we override it with old values. removed that and am re-testing now

Reply:
fixed